apps/keyboard/js/layouts/uk.js

```diff
@@ -0,0 +1,85 @@
+Keyboards.uk = {
+  label: 'Ukrainian',
+  shortLabel: 'Uk',
+  menuLabel: 'українська',
+  imEngine: 'latin',
+  types: ['text', 'url', 'email', 'password'],
+  basicLayoutKey: 'АБВ',
+  lang: 'uk',
+  alt: {
```

Comment:
This part should be moved into `pages` as the first page.
See gl.js for example.